src/components/IndividualReviewCard.jsx

```diff
@@ -25,7 +25,7 @@ const IndividualReview = () => {
   return (
     <div>
       <ul>
-        <li key={singleReview.review_id} className="reviewCard">
+        <li key={singleReview.review_id} className="IndReviewCard">
```

Comment:
The name of this file is misleading as this isn't a card, it's a whole article. As such it's not a list and instead of returning a `div` > `ul` > `li` it should return a `section` as it's a single piece of related content 👍